### PR TITLE
Added footer

### PR DESCRIPTION
**Pull request by Rishi-Mishra0704**
- Write acknowledgement to my peers in readme file
- Adjust the linters 
- Add footer/Contact me section in the project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -222,5 +222,33 @@ <h1 class="greet">
...
+            <input type="text" id="name" name="user_name" placeholder="Name" required/>
+          </li>
+          <li class="email-form">
+            <label for="mail"></label>
+            <input type="email" id="mail" name="user_email" placeholder="Email" required/>
+          </li>
+          <li class="msg-form">
+            <label for="msg"></label>
+            <textarea id="msg" name="user_message" placeholder="Message" required></textarea>
+          </li>
```
Comment (Shedrack-Sunday) [inline index.html]: - [ ] Based on [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md), The message field should have a max- length of 500 characters. please kindly fix that:
image

File: index.html [hunk 2]
```diff
@@ -222,5 +222,33 @@ <h1 class="greet">
...
+      <div class="contacts">Contact me</div>
+      <div class="contacts-txt">
+        If you have an application you
+        are intrested in developing, a feature
+        that you need to build or a project that needs
+        coding . I'd love to help with it 
+      </div>
+      <form method="post" action="https://formspree.io/f/{form_id}">
+        <ul class="contact-form">
+          <li class="name-form">
```
Comment (Shedrack-Sunday) [inline index.html]: - [ ] On submission, the form does not submit successfully. please follow the guide [here](https://formspree.io/html) to add the needed `id` and other parameters. Kindly confirm it doe that it successfully before submitting for the next review 

# Output:
image

File: index.html [hunk 3]
```diff
@@ -222,5 +222,33 @@ <h1 class="greet">
...
+            <label for="name"></label>
+            <input type="text" id="name" name="user_name" placeholder="Name" required/>
```
Comment (Shedrack-Sunday) [inline index.html]: - [ ] Based on the required features in the [Project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md), the name filed should have a `maxlength="30"` attribute. please do make the change 👍 

image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @Rishi-Mishra0704    👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing most of the features from the project requirement, but there are still some which aren't been addressed yet.



Highlights
✅All linters are checked
✅ Descriptive commit messages 
✅ Structure codebase 
Required Changes ♻️

- [ ] I must commend you for adding the form as suggested 👏  , but presently, there's a bug that doesn't get the form to submit the input text. Check the comment section for a more detailed review.
- [ ] Some major requirements for the form field are not yet met. I've highlighted them in the comment section as well. do make the changes, and you can get approved and move to the net project 💯 

_check the comment section for a detailed review_

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.


Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.


Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Rishi-Mishra0704** [comment]
-  Add a max length of 30 to input name in contact form 
-  Add a max length of 500 to input Message in contact form 
-  Fix the form submission 
- Adjust styling for contact form

**Review by Olamarx — APPROVED** [review]
Hi @Rishi-Mishra0704,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters passed ✔️
- form submission ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**